types/src/client.rs

```diff
@@ -132,3 +134,13 @@ impl<Notif> Drop for Subscription<Notif> {
 		let _ = self.to_back.send(FrontToBack::SubscriptionClosed(id)).now_or_never();
 	}
 }
+
+impl<Notif> Drop for NotificationHandler<Notif> {
```

Comment:
@niklasad1 Can you take a look at my drop implementation / follow this code path?

Reply:
it looks good